arithmetization/src/main/java/net/consensys/linea/plugins/rpc/RequestLimiter.java

```diff
@@ -36,7 +38,7 @@ public RequestLimiter(int concurrentRequestsCount) {
   public <T extends PluginRpcRequest, R> R execute(T request, Function<T, R> processingFunc) {
     if (!semaphore.tryAcquire()) {
       throw new PluginRpcEndpointException(
-          RpcErrorType.INVALID_REQUEST,
+          UNAVAILABLE_EXCEPTION,
```

Comment:
How about this
```suggestion
          RpcErrorType.UNKNOWN,
```

Reply:
Using a dedicated error allows to pass an error code that the coordinator can use to customize its error handling.